src/main/java/core/AbstractPlayer.java

```diff
@@ -30,6 +32,7 @@ public final int getPlayerID() {
     public final AbstractForwardModel getForwardModel() {
         return forwardModel;
     }
+    public final void setForwardModel(AbstractForwardModel fm) {forwardModel = fm;}
```

Comment:
Is this used anywhere? This was avoided for a similar reason as the player ID, to avoid having those settable from outside the package and let the core code handle things correctly.

Reply:
No - it's not used anywhere - I'll put this back to package-view.

(I think I may have used it on a test case at some point, and then decided against. I can also see future developments that might use this; if for example we have a player that learns a forward model, or sets up an opponent model that uses a simplified one...but that's way in the future.)

Reply:
Yeah I can see that happening, but we can deal with it at that point :D